curriculum/challenges/english/02-javascript-algorithms-and-data-structures/object-oriented-programming/remember-to-set-the-constructor-property-when-changing-the-prototype.md

```diff
@@ -11,11 +11,13 @@ dashedName: remember-to-set-the-constructor-property-when-changing-the-prototype
 There is one crucial side effect of manually setting the prototype to a new object. It erases the `constructor` property! This property can be used to check which constructor function created the instance, but since the property has been overwritten, it now gives false results:
 
 ```js
-duck.constructor === Bird; // false
-duck.constructor === Object; // true
-duck instanceof Bird; // true
+duck.constructor === Bird;
+duck.constructor === Object;
+duck instanceof Bird;
 ```
 
+In order, these would return `false`, `true`, and `true`.
```

Comment:
I am wondering when we have a sequence of expressions, if it would be better to say:
> In order, the above expressions would evaluate to `false`, `true`, and `true`.

There are a few above this one for this commit that I did not highlight.